config-object-store/src/main/java/org/hypertrace/config/objectstore/ConfigObject.java

```diff
@@ -8,4 +8,10 @@ public interface ConfigObject<T> {
   Instant getCreationTimestamp();
 
   Instant getLastUpdatedTimestamp();
+
+  String getCreatedByEmail();
+
+  String getLastUserUpdateEmail();
```

Comment:
Do you nee the timestamp with this?

Reply:
In case it wasn't clear - I meant that it's currently missing. From a readability perspective, it might be easier to reorder the field so they appear in pairs:

```
  Instant getCreationTimestamp();
  String getCreatedByEmail();

  Instant getLastUserUpdateTimestamp();
  String getLastUserUpdateEmail();

  Instant getLastUpdateTimestamp();
  String getLastUpdateEmail();
```